src/AWS.Deploy.Orchestration/AWS.Deploy.Orchestration.csproj

```diff
@@ -7,10 +7,6 @@
     <RootNamespace>AWS.Deploy.Orchestration</RootNamespace>
   </PropertyGroup>
 
-  <ItemGroup>
```

Comment:
I added a project reference to `AWS.Deploy.Recipes.CDK.Common` so this reference is no longer needed.